docs/reuse/tutorial/setup_edge.rst

```diff
@@ -74,8 +75,9 @@ If successful, the terminal will output ``microk8s is running``
 along with a list of enabled and disabled add-ons.
 
 Juju is required to deploy the |12FactorApp| application.
-Install Juju using the ``3.6/stable`` track, and bootstrap a
-development controller:
+We'll install Juju using the ``3.6/stable`` track; since the snap is
```

Comment:
```suggestion
We'll install Juju using the ``3.6/stable`` track. Since the snap is
```

```suggestion
We'll install Juju using the ``3.6/stable`` track -- since the snap is
```

I find semicolons visually noisy and that they can lead many contributors astray. I'm considering a rule in Starcraft docs to avoid them. What do you think?